Makefile

```diff
@@ -121,6 +131,8 @@ tag_major_minor:
 	docker tag $(NAME)/node-base:$(VERSION) $(NAME)/node-base:$(MAJOR)
 	docker tag $(NAME)/node-chrome:$(VERSION) $(NAME)/node-chrome:$(MAJOR)
 	docker tag $(NAME)/node-firefox:$(VERSION) $(NAME)/node-firefox:$(MAJOR)
+	docker tag $(NAME)/node-opera:$(VERSION) $(NAME)/node-opera:$(MAJOR)
+	docker tag $(NAME)/node-phantomjs:$(VERSION) $(NAME)/node-phantomjs:$(MAJOR)
```

Comment:
Why is `node-phantomjs` here? This image was deprecated a while ago.

Reply:
Deleted